src/api/hooks/usePaymentRequests.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { PaymentRequestPageResponse, PaymentRequest } from "../types/ApiResponses";
+import { getPaymentRequests } from '../PaymentRequestClient';
+
+export const usePaymentRequests = (apiUrl: string, token: string, page: number, pageSize?: number) => {
+    
+  const [paymentRequests, setPaymentRequests] = useState<PaymentRequest[]>([]);
+  const [pageNumber, setPageNumber] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+
+    useEffect(() => {
+      
+      const fetchPaymentRequests = async () => {
+        
+        const response = await getPaymentRequests(apiUrl, token, page, pageSize);
```

Comment:
I won't encourage to use use the methods of a specific client, in this case `PaymentRequestClient`, as we expect people to use  the methods of this from the general wrapper, `MoneyMoovApiClient`.

I imagine the usage to be more more like (syntax could be wrong, this is just to demostrate):
```
const moneyMoov = new MoneyMoovClient("eyJhbGc...") // Use token

const response = await moneyMoov.getPaymentRequests(page, pageSize);
```